toonz/sources/toonz/filmstrip.cpp

```diff
@@ -1958,6 +1989,10 @@ void Filmstrip::load(QSettings &settings) {
   m_showNavigator = navigator == 1;
   m_frames->setNavigator(m_showNavigator);
 
+  UINT separatorLine  = settings.value("separatorLine", 1).toUInt();
```

Comment:
> OT doesn't load the setting if set as false on already created rooms when it adds the settings to the ini file

This is because you set the second argument of `settings.value()` to `1`.
The second argument of this function is `default value` when the the setting doesn't exist.
I think it should be like `(m_showSeparatorLine?1:0)` .

Reply:
That was it, thank you!